init/common.js

```diff
@@ -202,6 +204,19 @@ const createProject = async (interestId, userId, project, addTeam=true) => {
   });
 };
 
+const createOpportunity = async (interestId) => {
+  let existing = await strapi.entityService.findMany("api::opportunity.opportunity", {filters: {title: config.opportunity.title}});
+  existing = existing[0];
```

Comment:
Since we only need the first opportunity, we can use `strapi.entityService.findOne`.

Reply:
@chungthuang  Thank you for your review. findOne with filteres not working. https://docs.strapi.io/dev-docs/api/entity-service/filter. Is there any alternate way to filter by title?